core/settings.py

```
@@ -29,7 +44,7 @@
 SECRET_KEY = os.environ.get('SECRET_KEY', 'django-insecure-dev-key-for-local-testing')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = os.environ.get('DEBUG', 'True').lower() == 'true'
+DEBUG = env_bool('DEBUG', True)
```

Ruff (0.15.14)

[warning] Boolean positional value in function call

(FBT003)